exporter/exporterhelper/internal/persistent_storage_test.go

```diff
@@ -217,7 +216,7 @@ func TestPersistentStorage_CorruptedData(t *testing.T) {
 				newPs.mu.Lock()
 				defer newPs.mu.Unlock()
 				return newPs.size() == c.desiredQueueSize && len(newPs.currentlyDispatchedItems) == c.desiredNumberOfDispatchedItems
-			}, 500*time.Millisecond, 10*time.Millisecond)
+			}, 1000*time.Millisecond, 10*time.Millisecond)
```

Comment:
If the timeout is the suspect I think you can be more generous with this, e.g. 5 seconds or something like that. It should typically very fast so the larger value of the timeout is not going to affect the normal runs.

Reply:
Good point, I have set timeout to 5 seconds consistently across the test